client/components/account-status/style.scss

```diff
@@ -12,7 +12,7 @@
 
 	// on a smaller viewports controls section should move in front of the
 	// status chip
-	@media only screen and ( max-width: 400px ) {
+	@media only screen and ( max-width: 390px ) {
```

Comment:
Although it must have taken a lot of effort I realize, but this can be done a better way without using media queries. 😅 

The tried doing it only using flex, can you please check the commit: https://github.com/Automattic/woocommerce-payments/commit/34bc17666e8f4edc9f61fd8b408c62ccd5603ee0

Branch: `add/1409-account-details-info`